test/functional/test_site_expressive_layout/_markbind/layouts/default/page.md

```diff
@@ -0,0 +1,14 @@
+<variable name="layoutVar">Variable from layout</variable>
+<import importedVar from="imported.md"/>
+
+{{ layoutVar }}
+{{ importedVar }}
```

Comment:
This variable doesn't seem to work? Unless I missed it, I don't see it in the output. :X

Reply:
Thank you so much for the review :smile:! 

I fixed this by adding a namespace. Seems like without assigning a namespace importedVar wouldn't work, this is the case even for current release 2.9.0.

Sidenote: I don't think this is the intended behaviour but could investigate in a separate issue and PR.